### PR TITLE
Site updates: New WhitePaper

### PR DESCRIPTION
**Pull request by zprood**
## Done

Add Robot whitepaper takeover and landing page based on English site template.

### CODE REVIEW

File: templates/engage/robot-operating-system-choice-cn.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+</section>
+
+
+<section class="p-strip is-shallow">
+  <div class="row">
+    <div class="col-7">
+      <h3>关于白皮书</h3>
+        <p>随着人工智能和机器学习的发展，在人们所能想象的每个领域，包括大众消费、工业、医疗保健和零售行业，都能看到机器人的身影。 </p>
+
+        <div class="col-12 u-equal-height">
```
Comment (anthonydillon) [inline templates/engage/robot-operating-system-choice-cn.html]: Please replace `col-12` with `row`

File: templates/engage/robot-operating-system-choice-cn.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+          <div class="col-2">
+            <img src="https://assets.ubuntu.com/v1/5f75154f-bot.png" alt="Robot with a display">
+          </div>
+          <div class="col-9 u-vertically-center">
```
Comment (anthonydillon) [inline templates/engage/robot-operating-system-choice-cn.html]: Replace `col-9` with `col-10`

Comment (zprood) [reply templates/engage/robot-operating-system-choice-cn.html]: Done

File: templates/engage/robot-operating-system-choice-cn.html (hunk 1 above)
Comment (zprood) [reply templates/engage/robot-operating-system-choice-cn.html]: Done

**Comment by zprood** [comment]
@anthonydillon Can I merge the PR into master ?

**Comment by anasereijo** [comment]
Takeover on the homepage looks good 👍 

My only suggestion is to reduce the height of the hero on the landing page. At this moment it's the same height as the takeover so people might not notice they moved to a new page... We can remove the robot illustration too if that helps to reduce the height.

thank you

File: templates/engage/robot-operating-system-choice-cn.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+<style>
+  .p-takeover--robotics {
+    background-color: #262626;
+  }
+  @media (min-width: 768px) {
+    .p-takeover--robotics {
+      background-image: url('https://assets.ubuntu.com/v1/a9f785e5-circuits-together.svg'), linear-gradient(-44deg, #171717 0%, #181818 9%, #262626 34%, #2D2D2D 67%, #383838 88%, #2E2E2E 100%, #393939 100%);
+      background-color: #262626;
+      background-size: cover;
+      padding: 8rem 0 !important;
```
Comment (anthonydillon) [inline templates/engage/robot-operating-system-choice-cn.html]: Please remove this padding styling

File: templates/engage/robot-operating-system-choice-cn.html [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+{% block content %}
+
+<style>
+  .p-takeover--robotics {
+    background-color: #262626;
+  }
+  @media (min-width: 768px) {
+    .p-takeover--robotics {
+      background-image: url('https://assets.ubuntu.com/v1/a9f785e5-circuits-together.svg'), linear-gradient(-44deg, #171717 0%, #181818 9%, #262626 34%, #2D2D2D 67%, #383838 88%, #2E2E2E 100%, #393939 100%);
```
Comment (anthonydillon) [inline templates/engage/robot-operating-system-choice-cn.html]: Remove the background image

File: templates/engage/robot-operating-system-choice-cn.html [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+    }
+  }
+</style>
+
+<section class="p-strip p-takeover--robotics js-takeover">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1 class="p-takeover__title">选择机器人操作系统时要考虑的关键因素</h1>
+    </div>
+    <div class="col-6 u-align--center u-vertically-center">
```
Comment (anthonydillon) [inline templates/engage/robot-operating-system-choice-cn.html]: Remove `u-vertically-center`

File: templates/engage/robot-operating-system-choice-cn.html [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+</style>
+
+<section class="p-strip p-takeover--robotics js-takeover">
+  <div class="row u-equal-height">
+    <div class="col-6">
```
Comment (anthonydillon) [inline templates/engage/robot-operating-system-choice-cn.html]: Add `u-vertically-center`

File: templates/engage/robot-operating-system-choice-cn.html (hunk 3 above)
Comment (zprood) [reply templates/engage/robot-operating-system-choice-cn.html]: Done

File: templates/engage/robot-operating-system-choice-cn.html (hunk 4 above)
Comment (zprood) [reply templates/engage/robot-operating-system-choice-cn.html]: Done

File: templates/engage/robot-operating-system-choice-cn.html (hunk 5 above)
Comment (zprood) [reply templates/engage/robot-operating-system-choice-cn.html]: Done

File: templates/engage/robot-operating-system-choice-cn.html (hunk 6 above)
Comment (zprood) [reply templates/engage/robot-operating-system-choice-cn.html]: Done

**Comment by anasereijo** [comment]
@anthonydillon much better, thanks

**Pull request merged**